index-cn.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf8">
+<title>活动守则</title>
+<style>
+* {
+  font-family: "Microsoft YaHei",  "微软雅黑体", "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
Throwing my usual objection to hardcoding Chinese sans-serifs here too: It's usually better for the user agent to decide what to use for the user's browser. (Yes, Microsoft Edge does not use that awful SimSun serif thing for sans-serif like IE, so you will be fine, really.) Hardcoding fonts that don't have multiple weights while invoking a lighter weight of 200 is pretty bad.